### PR TITLE
new design for no alerts in alert-list, fixes #9721

### LINKED ISSUE #9721: [UX] Improve design for no alert list panel when no alerts are available
We now render a text in the alert list panel when its empty.

I think we can make it look better. @Ijin08 Would you please take a look?

Just create a new dashboard and add an alert list panel and should see it.

### PR DESCRIPTION
**Pull request by Ijin08**
Fixed indentation of code, so changes might be harder to spot.
Changes in module.html are on row 4 - 14.

### CODE REVIEW

**Comment by bergquist** [comment]
Could you please attach a screenshot for this? makes it easier to review :)

File: public/sass/pages/_alerting.scss [hunk 1]
```diff
@@ -132,6 +132,13 @@
   }
 }
 
+.no-alert {
```
Comment (torkelo) [inline public/sass/pages/_alerting.scss]: this should be in the _panel_alertlist.scss no? And maybe named something more specific

Comment (Ijin08) [reply public/sass/pages/_alerting.scss]: All the other alert-list classes are in the _alerting.scss. Maybe they should all be in the _panel_alertlist.scss? Will rename class.

**Comment by Ijin08** [comment]
skarmavbild 2017-11-02 kl 08 28 53

@bergquist 

My idea was to keep it clear and simple. And also similar to other panels. Maybe change the font styling to the same as graph/heatmap.

**Comment by torkelo** [comment]
was not very happy with this so did a rewrite here: 
https://github.com/grafana/grafana/commit/a433698ddf357289c8bf2e4bdf14bb641d003e4a

**Comment by torkelo** [comment]
no, here https://github.com/grafana/grafana/commit/cd95f5bcc20e77353ac28b0860546a60bb37b9b6

**Pull request merged**